docs/set_delegate.md

```diff
@@ -3,31 +3,44 @@ title: Delegation
 author: Simon Boissonneault-Robert
 ---
 
-# Examples demonstrating delegation for the various address types
+# Delegation and Baking/Staking
+
+## What is baking (staking) in Tezos?
+Staking is a concept of participation in forming blocks in any blockchain using the Proof-of-Stake consensus algorithm. In Tezos we call this term "Baking". Everyone who holds Tezos coins can produce, sign, and validate blocks to get rewards in proportion to their stake. 
+
+To become a baker, all you need to do is to run your own node with a baking software and keep it online and up to date. 
+
+## What is Delegation?
+Delegation is when you give your baking rights to another person (baker). This mechanism in Tezos allows users to participate in staking and receive Tezos rewards without running their own node.
 
 In Tezos, a delegation operation will set the delegate of an address.
 
 When the `Babylon/proto005` protocol amendment came into effect, it changed how delegation from KT1 addresses work. Calling the KT1's smart contract `do` method is required to set the delegate for a KT1 account.  The `do` method takes a lambda function, and it is the logic of this function that causes the desired delegation to happen.
 
-## Delegate from an implicit tz1 address
+## Delegate from an implicit address (`tz1` prefix)
+Taquito has two main methods that facilitate a delegate operation, `setDelegate()` and `registerDelegate()`
+
+The main difference between the 2 methods is that `registerDelegate()` will set the delegate to the current address while`setDelegate()` will set the delegate to another address (baker).
```

Comment:
drop "main"

Reply:
add a space :  while`setDelegate()`  ->  while `setDelegate()`

Reply:
putting 'main' because they have a synonymous purpose in conducting a delegate operation.